tests/test_diff_tally/test_diff_tally.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+import glob
+import os
+import pandas as pd
+try:
+    from StringIO import StringIO
+except:
+    from io import StringIO
+import sys
+sys.path.insert(0, os.pardir)
+from testing_harness import PyAPITestHarness
+from openmc import Filter, Mesh, Tally, TalliesFile, Summary, StatePoint
+#from openmc.statepoint import StatePoint
+from openmc.source import Source
+from openmc.stats import Box
```

Comment:
From [PEP8](https://www.python.org/dev/peps/pep-0008/#imports): Imports should be grouped in the following order:

1. standard library imports
2. third party imports
3. local application specific imports

You should put a blank line between each group of imports.